### ISSUE TITLE
Current version change to 3.6 Object detect issue

### ISSUE THREAD

**Issue by hagsac** [stereolabs/zed-examples#436]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues.
- [x] This issue is not a question, feature request, or anything other than a bug report directly related to this project.

### Description

In the previous CUDA 11.1, 3.5.X SDK is normally used,
After installing CUDA 11.4, and 3.6 SDK,
There is a problem in the dialog history of the ZED SDK and Graphics Card.

After changing to CUDA 10.2 and installing 3.6SDK, the dialog is normal as shown in the figure below, but Object Detect does not work properly.

### Steps to Reproduce

1. Install cuda 11.4
2. Install sdk 3.6
3. object detection example failed
4. zed sdk, graphics card problem occurred in diagnostic
4. Reinstall multiple times;
5. Failure
6. Install cuda 10.2
7. Install sdk 3.6
8. diagnostic normal
9. object detection example failed

### Expected Result

Normal operation.

### Actual Result

![image](https://user-images.githubusercontent.com/75370807/138324353-906f92d0-e7b8-43bc-8d31-1e01998acbe2.png)

![image](https://user-images.githubusercontent.com/75370807/138324751-30993059-b060-4ff8-a366-9708ea5b42ed.png)

### ZED Camera model

ZED2i

### Environment

```shell
win 10
Processor Intel(R) Core(TM) i7-8700 CPU @ 3.20 GHz 3.19 GHz
16.0 GB of installed RAM
System type 64-bit operating system, x64-based processor
rtx 2070

1.cuda 11.4 sdk 3.6,
2.cuda 11.2 sdk 3.6
```

**Comment by Murdism** [comment]
> Ok, so it's not about the version of CUDA then.
> 
> * Do you have this issue with all models?
> * Could you try running the Diagnostic tool with the new option:
> 
> First clean all remaining model related files:
> 
> ```
> /usr/local/zed/tools/ZED_Diagnostic -aic
> ```
> 
> Then try to optimize them:
> 
> ```
> /usr/local/zed/tools/ZED_Diagnostic -aio
> ```
> 
> * Could you run the sample in gdb to have more information about the stack (especially where it's crashing)?
> * Could you post the output of:
> 
> ```
> ldd /usr/local/zed/sl_ai.so
> ```
> 
> * Could you post your diagnostic file report?

Hello i have the same issue, 
# i am using ubuntu 18.04 with cuda 10.2 and ZED sdk 3.7.0 
# object detection sample causes segmentation fault.

-> here is a result /usr/local/zed/tools/ZED_Diagnostic -aio

Optimizing all AI models
Optimizing: MULTI CLASS DETECTION...
  [ZED][INFO] AI model not found, downloading
\[ZED][INFO] Done
[ZED][INFO] Done
 |[ZED][INFO] Please wait while the AI model is being optimized for your graphics card
 This operation will be run only once and may take a few minutes 
|Stack trace (most recent call last):
#11   Object "[0xffffffffffffffff]", at 0xffffffffffffffff, in 
#10   Object "/usr/local/zed/tools/ZED_Diagnostic", at 0x432ef9, in 
#9    Object "/lib/x86_64-linux-gnu/libc.so.6", at 0x7f2805237c86, in __libc_start_main
#8    Object "/usr/local/zed/tools/ZED_Diagnostic", at 0x41bc8d, in 
#7    Object "/usr/local/zed/tools/ZED_Diagnostic", at 0x434b35, in 
#6    Object "/usr/local/zed/tools/ZED_Diagnostic", at 0x46d3f2, in 
#5    Object "/usr/local/zed/tools/ZED_Diagnostic", at 0x46c9ca, in 
#4    Object "/usr/local/zed/lib/libsl_ai.so", at 0x7f27febfa827, in 
#3    Object "/usr/local/zed/lib/libsl_ai.so", at 0x7f27febfdf30, in 
#2    Object "/usr/local/zed/lib/libsl_ai.so", at 0x7f27fec3166e, in 
#1    Object "/usr/local/zed/lib/libsl_ai.so", at 0x7f27fec2dc2a, in 
#0    Object "/usr/local/zed/lib/libsl_ai.so", at 0x7f27fec29a34, in 
Segmentation fault (Address not mapped to object [0x8])
Segmentation fault (core dumped)


Running ldd /usr/local/zed/sl_ai.so shows:
ldd: /usr/local/zed/sl_ai.so: No such file or directory
